docs/sources/reference/components/otelcol/otelcol.receiver.influxdb.md

```diff
@@ -0,0 +1,114 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/otelcol/otelcol.receiver.influxdb/
+aliases:
+  - ../otelcol.receiver.influxdb/ # /docs/alloy/latest/reference/otelcol.receiver.influxdb/
+description: Learn about otelcol.receiver.influxdb
```

Comment:
```suggestion
description: Learn about otelcol.receiver.influxdb
```

New topics don't need an alias. The alias is only used for moved/renamed topics.